docs/custom-metric-filter.md

```diff
@@ -0,0 +1,195 @@
+# ACM Custom Metric Filtering
+
+TL;DR: This file provides guides over adjusting the custom metrics which Config Sync exports to Cloud Monitoring, and workarounds for enabling/disabling
+the custom 
+
+## Background
+
+When ACM is installed on a GKE cluster where
+[default service accont](http://cloud/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform)
+is available, or on a GKE cluster that has
+[workload identity](http://cloud/kubernetes-engine/docs/how-to/workload-identity)
+enabled and
+[IAM is correctly setup](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync#custom-monitoring),
+ACM will try to apply the Open Telemetry Collector config map that contains
+[Googlecloud exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter)
+to write metrics into Cloud Monitoring. The default configuration contains a
+bare minimum list of metrics, but some customers might want to configure
+different backends or fine tune the metric list for better monitoring solutions.
+This doc serves as a workaround for users to opt out of custom monitoring, or
+adjust metric filtering.
+
+Note: this playbook refers to Anthos Config Management (ACM), but the same
```

Comment:
I think you can probably just drop this paragraph and change everything to refer to Config Sync, since this is the OSS docs for Config Sync.

Reply:
Done